docs/workloads/gcp/wordpress-gcp.md

```diff
@@ -17,115 +17,121 @@ You should have a `crossplane-gcp-provider-key.json` file on your local filesyst
 
```

Comment:
it would be good to have consistency across our workload examples.  Currently, we have the `examples/wordpress` dirs that all the docs use, and also the `examples/workloads` dir that only has GCP.  In this PR, you have updated the workloads documentation to use this GCP workload example.

Can you please open an issue to also update the AWS and Azure workloads documentation to use corresponding `examples/workloads` examples that need to be created?  Then we can probably remove the old `examples/wordpress` dirs completely and standardize our examples and docs only on `examples/workloads`.

Reply:
There is also some inconsistency between directories that are called `gcp` vs `example-gcp`.

Reply:
#523 is the Azure and AWS issue.